pkg/storageincentives/agent.go

```diff
@@ -323,12 +333,23 @@ func (a *Agent) play(ctx context.Context) (uint8, []byte, error) {
 
 	storageRadius := a.radius.StorageRadius()
 
+	// true if frozen
+	isFrozen, err := a.redistributionStatuser.IsOverlayFrozen(ctx)
+	if err != nil {
+		a.logger.Info("error checking if stake is frozen", "err", err)
+	}
+	a.nodeState.isFrozen(false)
+	if isFrozen {
+		a.nodeState.setLastFrozenRound(round)
```

Comment:
set should be capital S for consistency :)

Reply:
Let me make Initial capital for functions which are being used in agent.go.